test/cmdlineTests/standard_immutable_references/output.json

```diff
@@ -0,0 +1,2 @@
+{"contracts":{"a.sol":{"A":{"evm":{"deployedBytecode":{"immutableReferences":{"a.sol:A.x (3)":[77]},"linkReferences":{},"object":"bytecode removed","opcodes":"opcodes removed","sourceMap":"0:96:0:-:0;;;;5:9:-1;2:2;;;27:1;24;17:12;2:2;0:96:0;;;;;;;;;;;;;;;;12:1:-1;9;2:12;38:56:0;;;:::i;:::-;;;;;;;;;;;;;;;;;;;;72:7;90:1;83:8;;38:56;:::o"}}}}},"errors":[{"component":"general","formattedMessage":"a.sol: Warning: Source file does not specify required compiler version!
```

Comment:
``"a.sol:A.x (3)":[77]``
I'm not sure... maybe AST IDs only would be better... or just plain identifiers ``x``... but it'll be weirder with multi-slot types - I'll push another example with function pointers....